http/src/main/resources/init-resources/msdsvmcontent/ms_dsvm_script_v5_cat.sh

```diff
@@ -23,7 +23,7 @@ LISTENER_DOCKER_IMAGE=$8
 
 # Define environment variables for Jupyter Server customization
 
-SERVER_APP_PORT=8080
+SERVER_APP_PORT=8888
```

Comment:
any particular reason why we need to change port?

Reply:
Yes. RelayListener works on 8080. Since we introduce --network host it interferes with the the Jupyter port which is also 8080.